src/stringobj.c

```diff
@@ -415,10 +415,14 @@ static Obj FuncREVNEG_STRING(Obj self, Obj val)
 Obj NEW_STRING ( Int len )
 {
   Obj res;
-  if (len < 0)
-       ErrorQuit(
-           "NEW_STRING: Cannot create string of negative length %d",
-           (Int)len, 0L);
+  if (len < 0) {
+      ErrorQuit("NEW_STRING: Cannot create string of negative length %d",
```

Comment:
Minor consistency remark: most other error messages of this style continue in lower case after the colon:
```suggestion
      ErrorQuit("NEW_STRING: cannot create string of negative length %d",
```
Now, I realize that you only tweaked an existing message here, but you also added new errors which then of course should be consistent with this one, too...

Reply:
Looking at this function again inspired me to write PR #3528